library/alloc/src/sync/tests.rs

```diff
@@ -101,6 +101,45 @@ fn try_unwrap() {
     assert_eq!(Arc::try_unwrap(x), Ok(5));
 }
 
+#[test]
+fn unwrap_or_drop() {
+    // FIXME: Is doing this kind of loop reasonable? I tested `Arc::try_unwrap(x).ok`
+    // and it makes this kind of assersion fail in roughly every second run somewhere
+    // between 1000 and 5000 iterations; I feel like doing a sinlge iteration is too
```

Comment:
*single

Reply:
Thanks for those corrections, wrote that paragraph rather quickly, I was planning on removing this comment before this is merged anways, once I got an answer to that question.